src/components/createEventPageComponents/CreateEvent.jsx

```diff
@@ -50,18 +80,16 @@ function CreateEvent() {
                         </div>
                         <div className='col-span-6 sm:col-span-3'>
                             <label
-                                htmlFor='category'
+                                htmlFor='Interests'
                                 className='text-sm font-medium text-gray-900 block mb-2'
                             >
-                                Category
+                                Interests
                             </label>
-                            <input
-                                type='text'
-                                name='category'
-                                id='category'
-                                className='shadow-sm bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-cyan-600 focus:border-cyan-600 block w-full p-2.5'
-                                placeholder='e.g ..sustaibility'
-                                required
+                            <MultiSelect
```

Comment:
can we limite the number of selections with this componenet per example up to 5 max

Reply:
yes we can, although it's very difficult to predict the range of interests that an event can fall into.

Reply:
i get your point, my intentions is just to limit and the user takes the limit underconsideration that all